modules/database/src/Database.ts

```diff
@@ -296,7 +298,7 @@ export default class DatabaseModule extends ManagedModule<void> {
   async setSchemaExtension(call: CreateSchemaExtensionRequest, callback: SchemaResponse) {
     try {
       const schemaName = call.request.extension.name;
-      const extOwner = (call.metadata!.get('module-name')![0] as string) as string;
+      const extOwner = call.metadata!.get('module-name')![0] as string as string;
```

Comment:
Not sure why that double cast was previously there in the first place, but a single one should suffice.